osu.Game.Rulesets.Mania/Beatmaps/StageDefinition.cs

```diff
@@ -16,10 +17,18 @@ public struct StageDefinition
         public int Columns;
 
         /// <summary>
-        /// Whether the column index is a special column for this stage.
+        /// Get the <see cref="ColumnType"/> for a given column index.
         /// </summary>
         /// <param name="column">The 0-based column index.</param>
-        /// <returns>Whether the column is a special column.</returns>
-        public bool IsSpecialColumn(int column) => Columns % 2 == 1 && column == Columns / 2;
+        /// <returns>The type of the column.</returns>
+        public ColumnType TypeOfColumn(int column)
```

Comment:
Prefer `GetColumnType`.

I'm not 100% on this being here due to skin-specific special styles, but let's go with it for now.